wp-content/themes/thinkery/lib/output.php

```diff
@@ -148,15 +148,6 @@ function genesis_sample_css() {
 		'
 	: '';
 
-	$css .= ( has_custom_logo() && $logo_padding && ( 1 < $logo_effective_height ) ) ? sprintf(
-		'
-		.wp-custom-logo .title-area {
-			padding-top: %spx;
-		}
-		',
-		$logo_padding + 5
-	) : '';
-
```

Comment:
The settings in this file can be rather pesky at times. Great call just removing this from the get go, not sure if we wanna do the same on some of the CSS defaults values such as `$color_accent`, `$color_link` or `$color_primary`, but it may be worth considering.

Reply:
Yeah, I'm going with a remove as needed approach for now. Earlier on I toyed with actually using these settings (you'll see a 3rd color option in the optimizer). May end up scrapping that in exchange for a simplified pre-chosen color palette like we did for torque. Let's continue to think about that :)